cmd/system-probe/main.go

```diff
@@ -11,12 +11,15 @@ package main
 import (
 	"os"
 
+	"github.com/DataDog/datadog-agent/cmd/internal/rssshrinker"
 	"github.com/DataDog/datadog-agent/cmd/internal/runcmd"
 	"github.com/DataDog/datadog-agent/cmd/system-probe/command"
 	"github.com/DataDog/datadog-agent/cmd/system-probe/subcommands"
 )
 
 func main() {
+	rssshrinker.Setup()
```

Comment:
Do we agree that this is only to make the RSS metric look better than to actually reduce the actual memory used ?

Reply:
I think everyone agrees with that (although maybe this should be clearer on the PR description and in code comments), we're just releasing what's not used to the OS, but of course (and unfortunately !) it doesn't actually reduce used memory 😄